src/test/java/lesson1/SalaryTest.java

```diff
@@ -0,0 +1,31 @@
+package lesson1;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class SalaryTest {
+
+    private Salary salary;
+
+    @Before
+    public void setup() {
+        salary = new Salary();
+    }
+
+    @Test(expected = MyException.class)
+    public void itCalculateMonthSalary() throws MyException {
```

Comment:
The name bot actually tells what it tests.
I would recommend rename to -> `itThrowsExceptionWhenSalaryInHourIsNegative`